### PR TITLE
Add Common Scientific Units to Prelude

### PR DESCRIPTION
**Pull request by amadeusine**
This pull request adds common [units of length](https://en.wikipedia.org/wiki/Unit_of_length) among other scientific units.
- constants
	- speed of light in a vacuum (`c`) 
- energy
	- joules (J)
- microscopic distances
	- angstroms (Å)
	- microns (μm)
- astronomical distances
	- astronomical units (AU)
	- lightseconds (ls)
	- lightyears (ly)
	- parsecs (pc)
- _extremely scientific_ informal analogies
	- American football fields (equivalent to 100 Imperial yards)

### CODE REVIEW

File: lib/prelude.nat [hunk 1]
```diff
@@ -43,3 +43,25 @@ unit °C = celsius
...
+unit joule = kg * (m / s)^2
+unit J = joule
+
+unit angstrom = 100 pm
+unit ångström = angstrom
+unit Å = angstrom
+
+unit micron = μm
+
+c = 299792458
```
Comment (cuddlefishie) [inline lib/prelude.nat:56]: Shouldn't this be

```suggestion
c = 299792458 m/s
```
?

File: lib/prelude.nat [hunk 2]
```diff
@@ -43,3 +43,25 @@ unit °C = celsius
...
+
+unit astronomical_unit = 149597870700 m
+unit AU = astronomical_unit
+unit au = astronomical_unit
+
+unit lightsecond = c m
```
Comment (cuddlefishie) [inline lib/prelude.nat:62]: Given the changed definition of `c`, this should be 
```suggestion
unit lightsecond = c * 1 s
```
which seems a lot more sane to me :thinking:

File: lib/prelude.nat (hunk 1 above)
Comment (amadeusine) [reply lib/prelude.nat:56]: Forgot to change this before committing. Thanks for the catch 😅

File: lib/prelude.nat (hunk 2 above)
Comment (amadeusine) [reply lib/prelude.nat:62]: 👍

**Pull request merged**